src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const ul = document.querySelector('ul');
+
+function salaryToNumber(salary) {
+  return parseInt(salary.replace(/\$|,/g, ''));
+}
+
+function getEmployees(list) {
+  const employees = [];
+
+  for (const person of list.children) {
+    const salary = person.getAttribute('data-salary');
+    const position = person.getAttribute('data-position');
+    const age = person.getAttribute('data-age');
```

Comment:
You can also get these values by the `dataset` attribute:
```
const salary = person.dataset.salary;
```
Your solution is good, don't worry. It's just FYI